Source/SwiftyJSON.swift

```diff
@@ -82,8 +114,9 @@ public struct JSON {
 
      - returns: The created JSON
      */
-    public static func parse(_ string:String) -> JSON {
-        return string.data(using: String.Encoding.utf8)
+    @available(*, deprecated: 3.2, message: "Use instead `init(parseJSON: )`")
+    public static func parse(json: String) -> JSON {
```

Comment:
I would prefer clearer over cleaner : `parse(json: JSON)` and `parse(jsonString: String)`. :)